src/client/helpers/commentHelpers.js

```diff
@@ -14,6 +14,10 @@ function getCommentAnchor(comment) {
  * @returns the comment linked or null if not found
  */
 export function getLinkedComment(comments = []) {
+  if (typeof window === 'undefined') {
```

Comment:
nit - might be good to use `_.isUndefined(window)` so code `if window is undefined` ....